homeassistant/components/kaiterra/__init__.py

```diff
@@ -30,10 +30,12 @@
     PLATFORMS,
 )
 
+DEFAULT_NAME = "Kaiterra Device"
+
 KAITERRA_DEVICE_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_DEVICE_ID): cv.string,
-        vol.Required(CONF_TYPE): vol.In(AVAILABLE_DEVICE_TYPES),
+        vol.Optional(CONF_TYPE): vol.In(AVAILABLE_DEVICE_TYPES),
```

Comment:
This is no longer needed, but I kept it for backward compatibility.

Reply:
Maybe it is better to move this integration to the UI instead?

Reply:
Yeah, would be nice, but it will take much longer to implement and some users are already affected by broken redirect.

Reply:
True, but we actually don't accept changes to YAML configuration either, see:

ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Technically, this works around that, by leaving the configuration (even though unused). I'm kinda leaning towards thinking that isn't an acceptable solution.

Reply:
I can keep the config untouched for now if this helps and try to work on the config changes during the weekend (probably in separate PR?).

Reply:
I've tried to migrate to config flow, but unfortunately unsuccessfully. Any help would be welcome.

Reply:
If you have questions on development subjects, I would recommend joining our Discord server. There are development channels in there with lots of people that can help you answering any question regarding this.

Reply:
Yeah, I know, but I won't have much time in the next weeks to push this any further :/ 
Should the integration be removed instead? Since the current version is not working correctly?